src/code/z_camera_data.c

```diff
@@ -692,7 +770,7 @@ CameraModeValue sSetDoorCModeTargetData[] = {
 
 // Camera_Subj4 only reads one setting which is used for flags
 CameraModeValue sSetCrawlspaceModeNormalData[] = {
-    CAM_FUNCDATA_SUBJ4(0x0000, 2, 30, 10, 45, 0x3200),
+    CAM_FUNCDATA_SUBJ4(0, 2, 30, 10, 45, CAM_INTERFACE_FLAGS(CAM_SHRINKWINVAL_LARGE, CAM_HUD_ALPHA_2, 0)),
```

Comment:
`CAM_FUNCDATA_SUBJ4` doesn't line up with `Camera_Subj4` usage, I think the first value (0) is actually the `interfaceFlags` here

Reply:
Yeah, their crawlspaces code clearly has experimental code that never made it into the final game. The flags here is very weird. But the way it's read, yeah the first value can use the interface flags macro